Revolt/Api/UserSettingsStore.swift

```diff
@@ -338,9 +347,14 @@ class UserSettingsData {
                 mfaStatus: try await state.http.fetchMFAStatus().get()
             )
             
```

Comment:
This change was added due to an infinite loop in waiting for notifications. When logging into the official server or a 3rd party, the request would make it to the API, but the user would never get the "Accept Notifications" prompt.